standards/code-style.adoc

```diff
@@ -338,12 +342,68 @@ if (condition) {
 }
 ----
 
-This is __bad__:
-
 [source, kotlin]
 ----
 if (condition)
-    body()  // bad!
+    body()  // WRONG!
+----
+
+Braces are not required for `when` branches and `if` statement bodies if they meet BOTH
+the following conditions:
+
+. the conditional and the branch each contain at most a single predicate
+. it fits on a single line
+
+For `if` statements, there is an additional condition that there be no `else if` or `else`
+branches.
+
+[source, kotlin]
+----
+// Okay
+if (string.isEmpty()) return
+
+// Okay
+when (value) {
+    0 -> doSpecificThing()
+    1 -> doAnotherSpecificThing()
+	else -> doGenericThing()
```

Comment:
this is indented weirdly, I think?

Reply:
Nice catch. My text editor is trying to be smart and I didn't catch it.